src/mawpy/steps/incremental_clustering.py

```diff
@@ -266,13 +176,13 @@ def _get_locations_to_cluster_center_map(clusters_list: list[Cluster]) -> dict:
     """
     locations_to_cluster_center_map = {}
     for this_cluster in clusters_list:  # for each cluster in L
-        cluster_radius = int(1000 * this_cluster.radiusC())  # get radius of each cluster
+        cluster_radius = int(1000 * this_cluster.get_cluster_radius())  # get radius of each cluster
         cluster_center = [
```

Comment:
isn't this just `this_cluster.center`?

Reply:
cluster_radius is the max distance between any point in the cluster and the cluster center.